test/integration_tests/integrationtest_ros_scanner_node.cpp

```diff
@@ -131,7 +133,7 @@ int main(int argc, char* argv[])
   ros::init(argc, argv, "integrationtest_ros_scanner_node");
   ros::NodeHandle nh;
 
-  ros::AsyncSpinner spinner{ 2 };
+  ros::AsyncSpinner spinner{ 1 };
```

Comment:
Maybe remove at all?

Reply:
The `SubscriberMock subscriber` needs at least 1 spinning thread.